ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -126,6 +126,12 @@ metadata:
   name: kubevirt-hyperconverged
   namespace: openshift-cnv
 spec:
+  workloads:
+    nodePlacement:
+      nodeSelector:
+        beta.kubernetes.io/instance-type: c5n.metal
```

Comment:
this might cause problems for us in the future. It's not guaranteed that we'll always be using `c5n.metal` instances or that we'll even always be running this step registry on aws.

is there a reason why this nodeplacement improves things? if so, we can probably do it, but in a different way

Reply:
Well if I don't do this node placement, virt-handler starts on the control plane nodes, and since there is no nested virt, vms might not start. It might not matter, but I wanted virt-handler to only run on the metal instance